provider/source/src/transforms/mod.rs

```diff
@@ -39,15 +39,57 @@ impl CldrCache {
                         continue;
                     };
 
+                    if transform == "und-Ethi-t-und-latn-m0-beta_metsehaf-geminate" {
```

Comment:
Nit: We should have issues for all these with a `TODO` in the code. One big issue is _probably_ fine.